MediaBrowser.Controller/Lyrics/LyricResponse.cs

```diff
@@ -0,0 +1,22 @@
+#nullable disable
+
+using System.Collections.Generic;
+
+namespace MediaBrowser.Controller.Lyrics
+{
+    /// <summary>
+    /// LyricResponse model.
+    /// </summary>
+    public class LyricResponse
+    {
+        /// <summary>
+        /// Gets or sets Metadata.
+        /// </summary>
+        public IDictionary<string, string> Metadata { get; set; }
```

Comment:
Looking at wikipedia and several lyric sites, there appear to be 9 base metadata fields that people use. Some do use other fields, but they appear to be the exception rather than the rule.